orttraining/orttraining/python/training/model_desc_validation.py

```diff
@@ -0,0 +1,164 @@
+import cerberus
```

Comment:
After approving this PR, I realized that python build typing dataclasses packages may be more appropriate for the purpose.

Reply:
I will create a separate task to investigate how we can replace cerberus by typing/dataclasses